.github/workflows/next-build-image.yaml

```diff
@@ -244,3 +313,208 @@ jobs:
           if: ${{ env.REF_NAME_SHORT != 'next' && !startsWith(env.REF_NAME_SHORT,'next-') }}
           run: |
             docker buildx imagetools inspect ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:${{ steps.meta-release.outputs.version }}
+
+        - name: Wait for manifest propagation
+          if: env.HAS_QUAY_AUTH == 'true'
+          run: |
+            echo "Waiting for Quay to propagate multi-arch manifests..."
+            sleep 15
```

Comment:
only 15 seconds is enough? should this be a for loop with timeout and checks every few seconds?

what if wafter 15 seconds it hasn't propagagted due to some performance issue at quay or an outage?